### PR TITLE
Add organizations & update discord links

### PR DESCRIPTION
**Pull request by yfeng2824**
Description
- Add "Organizations" under ecosystem
- Update the discord links to https://discord.gg/4Jcw8MwEEv

### CODE REVIEW

File: website/docs/ecosystem/organizations.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+| Org name       | Github repo                      | Major contents                                    |
+| -------------- | -------------------------------- | ------------------------------------------------- |
+| Nervos Network | https://github.com/nervosnetwork | CKB-VM, RFCs, Moleculec-es                        |
+| Cell Studio    | https://github.com/ckb-cell      | RGB++                                             |
+| Spore Protocol | https://github.com/sporeprotocol | Spore protocol                                    |
+| Cryptape       | https://github.com/cryptape      | CKB Node Probe, Omnilock, CoBuild, Anyone-can-pay |
+| Magickbase     | https://github.com/Magickbase    | CKB Explorer, Neuron, CKB Testnet Faucet          |
+| Nervina-labs   | https://github.com/nervina-labs  | JoyID, CoTA                                       |
+| CKB-JS         | https://github.com/ckb-js        | Lumos, Kuai, CKB-SDK-JS                           |
+| .bit           | https://github.com/DIDHQ         | DID, Voty                                         |
```
Comment (RetricSu) [inline website/docs/ecosystem/organizations.md:19]: should be https://github.com/dotbitHQ ? the content can be  did-contracts , dotbit.js

**Comment by RetricSu** [comment]
can you add two more?

- https://github.com/godwokenrises
- https://github.com/axonweb3
- https://github.com/synapseweb3 ( forcerelay )

**Comment by yfeng2824** [comment]
> can you add 3 more?
> 
> * https://github.com/godwokenrises
> * https://github.com/axonweb3
> * https://github.com/synapseweb3 ( forcerelay )

Since axon/godwoken are currently suspended, we won't put them here

File: website/docs/ecosystem/organizations.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| -------------- | -------------------------------- | ------------------------------------------------- |
+| Nervos Network | https://github.com/nervosnetwork | CKB-VM, RFCs, Moleculec-es                        |
+| Cell Studio    | https://github.com/ckb-cell      | RGB++                                             |
+| Spore Protocol | https://github.com/sporeprotocol | Spore protocol                                    |
+| Cryptape       | https://github.com/cryptape      | CKB Node Probe, Omnilock, CoBuild, Anyone-can-pay |
+| Magickbase     | https://github.com/Magickbase    | CKB Explorer, Neuron, CKB Testnet Faucet          |
+| Nervina-labs   | https://github.com/nervina-labs  | JoyID, CoTA                                       |
+| CKB-JS         | https://github.com/ckb-js        | Lumos, Kuai, CKB-SDK-JS                           |
+| .bit           | https://github.com/dotbitHQ      | did-contracts , dotbit.js                         |
+| synapseweb3    | https://github.com/synapseweb3   | forcerelay                                        |
```
Comment (RetricSu) [inline website/docs/ecosystem/organizations.md:20]:
```suggestion
| ckb-ecofund    | https://github.com/ckb-ecofund   | ccc                                        |
```

**Pull request merged**